patches/tModLoader/Terraria/ModLoader/GlobalItem.cs

```diff
@@ -191,6 +191,34 @@ public abstract class GlobalItem : GlobalType<Item>
 		public virtual void ModifyResearchSorting(Item item, ref ContentSamples.CreativeHelper.ItemGroup itemGroup) {
 		}
 
+		/// <summary>
+		/// Allows you to choose if a given bait will be consumed by a given player
```

Comment:
Need to specify precedence here.

Reply:
done False trumps true, trumps null. Placed in the comments of both here and player hooks.